
uint8 brightness  # Set to zero for no change to brightness
int32 start  # Set to -1 for no change to start
int32 stop  # Set to -1 for no change to stop

# Effect
string effect_name
uint8 effect_intensity
uint8 effect_speed

# Colours
Colour primary_colour
Colour secondary_colour
Colour tertiary_colour
string palette_name
